blazor-toc.html

```diff
@@ -1489,6 +1489,9 @@
 					<li>
 						<a href="/blazor/datagrid/how-to/hide-detail-template-icon-when-record-details-has-no-record">Hide detail template icon when record details has no record</a>
 					</li>
+					<li>
+						<a href="/blazor/datagrid/how-to/expando-object-binding-with-edit-template">ExpandoObject binding with edit template</a>
```

Comment:
move this topic to editing section. https://blazor.syncfusion.com/documentation/datagrid/cell-edit-types#custom-editors-using-templatecell-edit-template